crates/cdk/src/publish.rs

```diff
@@ -33,6 +34,10 @@ pub struct PublishCmd {
 
     pub package_meta: Option<String>,
 
+    /// path to the ipkg file, used when --push is specified
+    #[arg(long)]
+    ipkg: Option<String>,
```

Comment:
This is only should be used in conjunction with `--push`

Reply:
Yes but we also allow both `--pack` and `--push` at the same time, in that case this would mean nothing since push will use the artifact from pack step